src/mod/xml_int/mod_xml_curl/mod_xml_curl.c

```diff
@@ -530,6 +538,10 @@ static switch_status_t do_config(void)
 			binding->cookie_file = switch_core_strdup(globals.pool, cookie_file);
 		}
 
+		if (unix_socket_path) {
+            binding->unix_socket_path = switch_core_strdup(globals.pool, unix_socket_path);
```

Comment:
is it wrongly indented or is just github showing it wrong ? same for some other lines.